modules/client/examples/05-encoders-decoders/13-addresslist-voting/04-update-plugin-settings.ts

```diff
@@ -10,54 +10,55 @@ Updates the settings of a given AddresslistVoting plugin.
 ### Encoding
 */
 
-import {
-  DaoAction,
-  VotingMode,
-  VotingSettings
-} from "@aragon/sdk-client";
+import { DaoAction, VotingMode, VotingSettings } from "@aragon/sdk-client";
 import { addresslistVotingClient } from "./index";
 
 // The action object for updating the plugin settings.
-const configActionPrarms: VotingSettings = {
+const params: VotingSettings = {
   minDuration: 60 * 60 * 24 * 2, // seconds
   minParticipation: 0.25, // 25%
   supportThreshold: 0.5, // 50%
   minProposerVotingPower: BigInt("5000"), // default 0
-  votingMode: VotingMode.STANDARD // default STANDARD, otherwise EARLY_EXECUTION or VOTE_REPLACEMENT
+  votingMode: VotingMode.STANDARD, // default STANDARD, otherwise EARLY_EXECUTION or VOTE_REPLACEMENT
 };
 
 const pluginAddress: string = "0x1234567890123456789012345678901234567890"; // the address of the AddresslistVoting plugin contract installed in the DAO
 
-const action: DaoAction = addresslistVotingClient.encoding.updatePluginSettingsAction(pluginAddress, configActionPrarms);
+const action: DaoAction = addresslistVotingClient.encoding
+  .updatePluginSettingsAction(pluginAddress, params);
 console.log({ action });
 
 /* MARKDOWN
 Returns:
 
 ```
-{
-  to: "0x1234567890...",
-  value: 0n,
-  data: Uint8Array[12,34,45...]
+{ action:
```

Comment:
If it's the only value, we might not need this

Reply:
There are multiple console logs in the example thats why l did it this way